.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,29 @@
+name: goreleaser
+
+on:
+  pull_request:
```

Comment:
This should not be there. We don't create a release when a pull_request is raised